### PR TITLE
[dynamo][hf_bigbird] Actually graph break on tensor.unsqueeze_/resize_

### PR DESCRIPTION
**Pull request by anijain2305**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #100071
* __->__ #99986

Currently, we return `unimplemented` w/o a graph break on seeing a x.unsqueeze_ when x is input. This essentially means we fall back to the original frame.

This PR actually graph breaks so that we can generate the continuation frame for the rest of the function. Instead of graph breaking at LOAD_ATTR, we delay the graph break to the actual CALL_FUNCTION, where its cleaner to graph break.

cc @soumith @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: torch/_dynamo/symbolic_convert.py [hunk 1]
```diff
@@ -1133,6 +1133,7 @@ def CALL_METHOD(self, inst):
         fn = self.pop()
         self.call_function(fn, args, {})
 
+    @break_graph_if_unsupported(push=0)
```
Comment (jansel) [inline torch/_dynamo/symbolic_convert.py:1136]: `push=0` doesn't sound right

**Review by jansel — CHANGES_REQUESTED** [review]

**Comment by anijain2305** [comment]
@jansel Made some changes. Can you PTAL?

**Review by jansel — APPROVED** [review]

File: torch/_dynamo/variables/tensor.py [hunk 2]
```diff
@@ -182,7 +182,8 @@ def var_getattr(self, tx, name):
         # It's hard to get resize_() on graph input work properly across
         # dynamo/aot/inductor, just fall back.
         if name in ("resize_", "unsqueeze_") and self.source is not None:
```
Comment (eellison) [inline torch/_dynamo/variables/tensor.py:184]: can you use the [inplace tag](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/tags.yaml#L3-L5) list ? `resize_` and `unsqueeze` is an incomplete list

Comment (jansel) [reply torch/_dynamo/variables/tensor.py:184]: This list should only be for metadata mutation, not all mutation.

Comment (eellison) [reply torch/_dynamo/variables/tensor.py:184]: Yea, sorry I linked to `inplace_vew`, which is metadata only. mistyped

**Pull request closed without merging**